pyccl/covariances.py

```diff
@@ -158,12 +165,15 @@ def sigma2_B_disc(cosmo, a=None, fsky=1., p_of_k_a=None):
         float or array_like: values of the projected variance.
     """
     status = 0
-    if a is None:
+    full_output = a_arr is None  # return sampling?
```

Comment:
What does the comment mean?

Reply:
I just realized, is this copied+pasted from `star`? I have no idea what I meant, I must have been thinking of food or something, we can just remove the comment.

Reply:
Done